e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -183,7 +187,7 @@ void constraintCachedNoViolations() throws IOException {
     assertEquals(constraintDefinition, cachedRun.constraintDefinition());
 
     // Check results
-    assertTrue(cachedRun.results().isEmpty());
+    assertTrue(!cachedRun.results().isEmpty());
```

Comment:
This check needs to be fleshed out to validate the cached results are correct

Reply:
I am now checking the results object